lib/filewriter.js

```diff
@@ -139,13 +139,13 @@ FileWriter.prototype = {
             function finish(err) {
                 if (guard && global.clearTimeout) clearTimeout(guard);
                 guard = null;
-                if (fd !== null) fs.close(fd, function(){});
+                if (fd !== null) try { fs.closeSync(fd) } catch (e) { }
```

Comment:
the change I pushed showed up in PR #5 ?  see https://github.com/andrasq/node-qfputs/pull/5/files